src/core/Form/Select/MultiSelect/MultiSelect/MultiSelect.tsx

```diff
@@ -163,6 +176,8 @@ interface InternalMultiSelectProps<T extends MultiSelectData> {
   tooltipComponent?: ReactElement;
   /** Ref object to be passed to the input element. Alternative to React `ref` attribute. */
   forwardedRef?: React.RefObject<HTMLInputElement>;
+  /** Props to pass to unordered list element, for example data-attributes */
```

Comment:
Do we need to or want to emphasize the data-attributes here?

Reply:
Good question. I didn't add any new example about usage of listProps/listItemProps either, since it's not very usual use case. I thought that mentioning data-attributes as usage example would indicate what the props are for. 🤔

Reply:
Yeah, I guess it's ok to mention the most common use case here.